installer.sh

```
@@ -0,0 +1,986 @@
+    info "Installing PM2..."
+    npm install -g pm2 &>/dev/null || err "PM2 install failed"
+
+    info "Starting panel temporarily with PM2..."
+    cd /var/www/panel
+    pm2 start npm --name "airlink-panel-temp" -- run start &>/dev/null
+    ok "Panel started temporarily"
+
+    # Create admin user via API (skip prompt if called from install_all)
+    if [ "$skip_config" = false ]; then
+        # Always use pre-collected variables (true) since we now collect them at the start
+        create_admin_user true || {
```

Shellcheck (0.11.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)